microsoft-edge/webdriver-chromium/index.md

```diff
@@ -52,12 +52,22 @@ To begin automating tests, use the following steps to ensure that the WebDriver
 
 The last component you must download is a language-specific client driver to translate your code \(Python, Java, C\#, Ruby, JavaScript\) into commands the Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  
 
-[Download the WebDriver language binding of your choice][SeleniumDownloads].  The Microsoft Edge team recommends [Selenium 4.00-alpha07][NugetPackagesSeleniumWebdriver400alpha07] or later, because it supports Microsoft Edge \(Chromium\).  However, you may control Microsoft Edge \(Chromium\) in all older versions of Selenium, including the current stable Selenium 3 release.  
+[Download the WebDriver language binding of your choice][SeleniumDownloads].  The Microsoft Edge team recommends [Selenium 4.0.0-beta2][NugetPackagesSeleniumWebdriver400beta02] or later, because it supports Microsoft Edge \(Chromium\).  However, you can control Microsoft Edge \(Chromium\) in all older versions of Selenium, including the current stable Selenium 3 release.  
 
 > [!IMPORTANT]
 > If you previously automated or tested Microsoft Edge \(Chromium\) using `ChromeDriver` and `ChromeOptions` classes, your WebDriver code does not run on Microsoft Edge Version 80 or later.  To solve the problem, update your tests to use the `EdgeOptions` class and download [Microsoft Edge Driver][MicrosoftDeveloperMicrosoftEdgeToolsWebdriver].  
 
-### Use Selenium 3  
+### Using Selenium 4
+
+Selenium 4 has built-in support for Microsoft Edge (Chromium).
```

Comment:
Thanks, this section looks better but I'm still concerned that it focuses on how **not** to use Selenium 4 instead of how to use it correctly. Could we provide a link here to Selenium's docs for installing the libraries?https://www.selenium.dev/documentation/en/selenium_installation/installing_selenium_libraries/

Reply:
@bwaldbaum  Done, ready for review